digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/dataimport/ImportDataApi.scala

```diff
@@ -109,14 +109,14 @@ class ImportDataApi(roadLinkService: RoadLinkService, val userProvider: UserProv
     csvDataImporter.getByIds(ids)
   }
 
-  //TODO check if this exist
-  post("/csv") {
-    if (!userProvider.getCurrentUser().isOperator()) {
-      halt(Forbidden("Vain operaattori voi suorittaa Excel-ajon"))
-    }
-    val csvStream = new InputStreamReader(fileParams("csv-file").getInputStream)
-    new MassTransitStopExcelDataImporter().updateAssetDataFromCsvFile(csvStream)
-  }
+//  //TODO check if this exist
```

Comment:
The `MassTransitStopExcelDataImporter` is commented. This can probably be deleted.

Reply:
I agree